lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -204,12 +206,12 @@ defmodule DbserviceWeb.StudentController do
         })
       end)
 
-      # Create a new enrollment record with the fetched group_id
+      # Create a new enrollment record with the fetched status_group_id
       new_enrollment_attrs = %{
         user_id: user_id,
         is_current: true,
         start_date: current_time,
-        group_id: group_id,
+        status_group_id: status_group_id,
```

Comment:
This won't work, I think. It should be group_id only, because ER has group_id as attribute.

Reply:
Correct. The key won't match

Reply:
Changing this